README.md

```diff
@@ -156,14 +156,14 @@ First the good news: you will _not_ have to migrate your keychain data when upgr
 
 Now the bad news: the Swift Valet API has slight differences from the Objective-C Valet API. You may have noticed a few of the differences in the sample code above, but here's a rundown of the changes that may affect you.
 
-1. Initializers have changed in both Swift and Objective-C. [See examples above](#Basic-Initialization).
-2. `VALSynchronizableValet` has been replaced by a `Valet`  (or `VALValet` in Objective-C) with an `.iCloud` flavor. [See examples above](#Sharing-Secrets-Across-Devices-with-iCloud).
+1. Initializers have changed in both Swift and Objective-C - both languages use class methods now, which felt more semantically honest (a lot of the time you're not instantiating a new Valet, you're re-accessing one you've already created). [See example usage above](#Basic-Initialization).
+2. `VALSynchronizableValet` (which allowed keychains to be synced to iCloud) has been replaced by a `Valet.iCloudValet()`  (or `[VALValet iCloudValet]` in Objective-C). [See examples above](#Sharing-Secrets-Across-Devices-with-iCloud).
 3. `setObject(_:forKey:)` has become `set(object:forKey:)` in Swift. The Objective-C API `-setObject:forKey:` remains the same.
 4. `setString(_:forKey:)` has become `set(string:forKey:)` in Swift. The Objective-C API `-setString:forKey:` remains the same.
 5. `SecureEnclaveValet` and `SinglePromptSecureEnclaveValet` data retrieval methods now return a single enum [SecureEnclave.Result](Sources/SecureEnclave.swift#L28) rather than using an `inout` boolean to signal whether a user cancelled. The Objective-C API remains the same.
 6. `migrateObjects(matching:)` and `migrateObjects(from:)` now both return a nonnull [MigrationResult](Sources/MigrationResult.swift#L24).
 7. `VALAccessControl` has been renamed to `SecureEnclaveAccessControl` (`VALSecureEnclaveAccessControl` in Objective-C). This enum no longer references `TouchID`; instead it refers to unlocking with `biometric` due to the introduction of Face ID.
-8. `Valet`, `SecureEnclaveValet`, and `SinglePromptSecureEnclaveValet` are no longer in the same inheritance tree. All three now inherit directly from `NSObject` and use composition to share code.
+8. `Valet`, `SecureEnclaveValet`, and `SinglePromptSecureEnclaveValet` are no longer in the same inheritance tree. All three now inherit directly from `NSObject` and use composition to share code. If you were relying on the subclassing in place before, 1) that might be a code smell 2) consider declaring a protocol for the shared behavior you were expecting to make your migration to Valet 3 easier.
```

Comment:
nit: s/subclassing in place before/subclass hierarchy? "in place before" is throwing me a little.

I love the suggestions though

Reply:
Haha that's how I got Cash compiling so quickly (the protocol)

Reply:
🚀